CMSgov/design-system
# [WNMGDS-711] Update doc site <head> for child ds

### Summary
- Uses `option.name` inside the doc site `<head>`
- Conditionally add the core ds tealium tag when its the core doc site being built
- Consistently use the same design system description

### How to test
- Run a child design system with this branch
- Ensure the title and description have been updated
- Ensure the tealium script is not included
